[ch28750] Added a 404 & 500 html page

emaciupe opened the pull request:
[ch28750]

domdinicola commented on django_api/etools_prp/templates/http_codes_base.html:
```diff
@@ -0,0 +1,75 @@
+            m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
+        })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+        // this is for the staging site only
+        ga('create', 'UA-70082239-1', 'auto');
```
is this the same ID as eTools???

emaciupe replied:
@domdinicola ah yes, I knew I forgot to ask something here, this is a copy-paste, do we need GA for PRP? or remove it?

domdinicola replied:
@robertavram  @ntrncic I think we should remove this (are we using matomo?)
maybe we will need to remove it also from eTools

domdinicola replied:
@emaciupe please remove them and use matomo

emaciupe replied:
resolved!

emaciupe commented:
@ntrncic @domdinicola Requires to configure the matomo settings for backend.

domdinicola commented:
@emaciupe do you think is possible to track informations about the user as well?

emaciupe commented:
> @emaciupe do you think is possible to track informations about the user as well?

@domdinicola I've added tracking for the user id. 

> @emaciupe do you think is possible to track informations about the user as well?

@domdinicola yes, I've added user id tracking, see the screenshot on [Card #28750](https://app.shortcut.com/unicefetools/story/28750/add-404-page-similar-to-etools)
